header.css

```diff
@@ -1,27 +1,31 @@
+
+
 header {
     width: 375px;
     margin:0px;
+    
 }
 
 #fishlogoblue {
     height: 40px;
     width: 54px;
     margin-bottom: 2px;
     margin-left:10px;
+    margin-right:27px;
```

Comment:
Some inconsistencies between space after `:` - some have, others don't. Consider using a style linter in the group, to ensure a consistent style :)